PukingPredator/Assets/Scripts/Player.cs

```diff
@@ -1,22 +1,67 @@
+using System;
+using System.Collections.Generic;
+using Unity.VisualScripting;
 using UnityEngine;
 
+public enum MovementState
```

Comment:
this will likely need to go beyond just "movement state" in the future, it makes more sense to just make it "PlayerState" in case it represents more